src/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -33,6 +33,26 @@ Object {
 }
 `;
 
+exports[`denormalize denormalizes recursive dependencies 1`] = `
+Object {
```

Comment:
My expected output would be:

```js
{
  title: "Weekly report",
  user: {
    role: 'manager',
    reports: [ 1 ]
  }
}
```

This is going too many levels deep and isn't properly keeping track of the fact that we've already come across report 1.